annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/BaseJsonldRest.java

```diff
@@ -488,8 +472,8 @@ protected ResponseEntity<String> updateAnnotation(String identifier, String anno
 	    String eTag = generateETag(updatedAnnotation.getGenerated(), WebFields.FORMAT_JSONLD, apiVersion);
 
 	    // serialize to jsonld
-	    JsonLd annotationLd = new AnnotationLdSerializer(updatedAnnotation);
-	    String jsonLd = annotationLd.toString(4);
+        annotationLdSerializer.setAnnotation(updatedAnnotation);
```

Comment:
annotationSerilizer is statefull, therefore a new object must be created for each request